hexrd/ui/indexing/fit_grains_options_dialog.py

```diff
@@ -206,6 +213,10 @@ def update_gui_from_config(self, config):
 
         indexing_config = HexrdConfig().indexing_config
         self.selected_material = indexing_config.get('_selected_material')
+        self.working_dir = indexing_config.get(
+            'working_dir', HexrdConfig().working_dir)
```

Comment:
Because `indexing_config['working_dir']` should be the parent directory, and it appears that `self.working_dir` is supposed to be the full path, we should set `self.working_dir` similar to this instead:

```
working_dir = indexing_config.get('working_dir', str(Path(HexrdConfig().working_dir).parent))
analysis_name = indexing_config.get('analysis_name', str(Path(HexrdConfig().working_dir).stem))
self.working_dir = str(Path(working_dir) / analysis_name)
```

Maybe we should rename `self.working_dir` to something like `self.spots_path` to disambiguate the fact that it is not the same thing as `indexing_config['working_dir']`.

Reply:
Sounds good